test/expr/unary/if_expr.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-typecheck-verify-swift -disable-availability-checking
+// RUN: %target-typecheck-verify-swift -disable-availability-checking -enable-experimental-concurrency
```

Comment:
Should this also use `// REQUIRES: concurrency` and `// REQUIRES: asserts`?

Reply:
Ah yeah, good catch! Probably ought to separate out the tests that need this